app/controllers/records_controller.rb

```diff
@@ -24,7 +24,7 @@ def update
   end
 
   def destroy
-    @record.destroy
+    current_record.destroy
```

Comment:
`current_record` is already called before the action. If you are not familiar with filters read [action controller's filters](http://guides.rubyonrails.org/action_controller_overview.html#filters)